WikiWatershed/model-my-watershed
# Add NLCD 2016 Edition Visual Layers

## Overview

Add the four new NLCD 2016 version visual layers to the Coverage Tab of the Layer Selector.

These were ingested via GDAL on EC2, using the script specified here: https://github.com/WikiWatershed/model-my-watershed/issues/3393#issuecomment-866179490

The old layer was renamed "National Land Cover Database Old 2011". /cc @darscott @SRGDamia1 @aufdenkampe for confirmation if this is appropriate, or if we should call it something else.

The layer selector was widened a bit to accommodate the lengthier layer names.

Connects #3394 

### Demo

![2021-06-28 11 28 40](https://user-images.githubusercontent.com/1430060/123663630-9408b700-d804-11eb-85ab-98b431c2cafd.gif)

The NLCD Analyze connects to the Old 2011 layer, this will be amended in #3395 

![image](https://user-images.githubusercontent.com/1430060/123664427-548e9a80-d805-11eb-9691-8383994b5cac.png)

## Testing Instructions

* Check out this branch and `./scripts/bundle.sh --debug --vendor`
* Go to [:8000/](http://localhost:8000/)
* Switch the layer selector to the Coverage (second) tab
  - [x] Ensure you see 4 new NLCD layers, and they work correctly
  - [x] Ensure the old NLCD layer is renamed
* Choose a shape to analyze
  - [x] Ensure the Land Analysis still uses the Old 2011 layer, and it turns on from the sidebar correctly
